external-table/src/pxfbridge.c

```
@@ -135,8 +135,8 @@ static void
 build_uri_for_read(gphadoop_context *context)
 {
 	resetStringInfo(&context->uri);
-	appendStringInfo(&context->uri, "http://%s/%s/%s/Bridge/",
-					 get_authority(), PXF_SERVICE_PREFIX, PXF_VERSION);
+	appendStringInfo(&context->uri, "http://%s/%s/v15/Bridge/",
```

why v15? shouldn't we use the new endpoints? The legacy endpoints should return an error (like the legacy `getFragments` endpoint).

---

yes, please rebase off master, use new endpoints and make old ones obsolete as part of this PR.